app/lib/efile/nj/nj_municipalities.rb

```diff
@@ -0,0 +1,54 @@
+module Efile
+  module Nj
+    class NjMunicipalities
+      class << self
+        def county_options
+          load
+          @municipalities_by_county.keys.sort
+        end
+
+        def find_name_by_county_and_code(county, code)
+          load
+          county_rows = @municipalities_by_county[county]
+          raise KeyError.new("Invalid county") unless county_rows
+          municipality = county_rows.find { |muni| muni.code == code }
+          raise KeyError.new("Invalid code") unless municipality
```

Comment:
[DUST] I might include "municipality" in this message, as "Invalid municipality code" or "Invalid municipality" as a good practice, but it's incredibly minor here